test/claims-status/components/appeals-v2/AppealsV2StatusPage.unit.spec.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import { mockData } from '../../../../src/js/claims-status/utils/helpers';
+import AppealsV2StatusPage from '../../../../src/js/claims-status/containers/AppealsV2StatusPage';
+
+describe.only('<AppealsV2StatusPage/>', () => {
+  const defaultProps = { appeal: mockData.data[0] };
+
+  it('should render', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    expect(wrapper.type()).to.equal('div');
+  });
+
+  it('should render an unexpanded <Timeline/>', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const timeline = wrapper.find('Timeline').dive();
+    const expander = timeline.find('Expander');
+    const pastEvents = timeline.find('PastEvent');
+    expect(expander.length).to.equal(1);
+    expect(pastEvents.length).to.equal(0);
+  });
+
+  it('should pass down events as props to Timeline', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const pastEvents = wrapper.find('Timeline').props().events;
+    expect(pastEvents.length).to.equal(defaultProps.appeal.attributes.events.length);
+  });
+
+  it('should render a <CurrentStatus/> with title and description', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const statusProps = wrapper.find('CurrentStatus').props();
+    // Testing against a specific string would require duplicating component
+    // and util code so here we're just testing that both props exist
+    expect(statusProps.title).to.exist;
+    expect(statusProps.description).to.exist;
+  });
+
+  it('should render an <Alerts/> with alerts', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const alerts = wrapper.find('Alerts');
+    const alertsProps = alerts.props();
+    expect(alerts.length).to.equal(1);
+    expect(alertsProps.alerts.length).to.equal(defaultProps.appeal.attributes.alerts.length);
+  });
+
+  it('should render a <WhatsNext/> with nextEvents', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const whatsNext = wrapper.find('WhatsNext');
+    const nextProps = whatsNext.props();
```

Comment:
Maybe not worth changing, but when I saw `nextProps`, I first thought it was referring to `<AppealsV2StatusPage>`'s next props, not `<WhatsNext>`'s props. Took me more time to figure out whether or not it was worth mentioning than it did to figure out what it was referring to.

My mild OCD wouldn't let me move on without at least mentioning it, but don't feel obligated to change anything.

Reply:
Funny you mention that, I thought this as well and figured I was the only one who'd notice. Changed.